core/src/main/java/org/fao/geonet/kernel/GeonetworkDataDirectory.java

```diff
@@ -162,6 +162,9 @@ private Path lookupProperty(JeevesServlet jeevesServlet, ServiceConfig handlerCo
                     if (jeevesServlet != null) {
                         value = jeevesServlet.getInitParameter(keyToUse);
                     }
+                    if ( (value == null) && (jeevesServlet != null) && (jeevesServlet.getServletContext() != null) ){
```

Comment:
Just small refactor, to move the code inside the if from line 162, as already checks the condition`jeevesServlet != null`:

```
if (jeevesServlet != null) {
  value = jeevesServlet.getInitParameter(keyToUse);
  if ( (value == null) && (jeevesServlet.getServletContext() != null) ) {
    ...
  }
}

```

Reply:
moved